cmd/skaffold/app/cmd/build_test.go

```diff
@@ -48,6 +48,40 @@ func (r *mockRunner) Stop() error {
 	return nil
 }
 
+func TestTagFlag(t *testing.T) {
+	mockCreateRunner := func(config.SkaffoldOptions) (runner.Runner, *latest.SkaffoldConfig, error) {
+		return &mockRunner{}, &latest.SkaffoldConfig{}, nil
+	}
+
+	tests := []struct {
```

Comment:
Since there's a single test, you should be able to inline all those values.

Reply:
I modified code and CI passed again.